requirements/base.in

```diff
@@ -13,4 +13,4 @@ requests==2.9.1
 pypng
 simplejson
 webob
-XBlock==1.1.1
+XBlock==1.2.6
```

Comment:
I have also updated this version. Let me know if its fine ?

Reply:
I would just unpin this in base.in since this now works with the latest version. and run `make upgrade` to update the txt files.

Reply:
@feanil I just updated the acid-block hash in this PR. Updating Xblock and make upgrade making things very complicated.